term_proposal/evan11401/project.md

```diff
@@ -0,0 +1,84 @@
+# High Dynamic Range Imaging using Multiple Exposure Images with Tone Mapping
+![img1](./img1.jpg)
+## Motivation
+The real world is high dynamic range, but the camera is limited dynamic range. In case to get the image that looks more like the real world image, high dynamic range imaging (HDRI) is invented.
+
+HDRI is widely used in the imaging and photography industry, such as the APPLE iPhone's camera also use this to get a better image.
+
+## Target
+I have read some tutorial to teach people how to implement the HDRI algorithm, but most of the function is provided by the OpenCV library.
+
+I know the library is powerful, but I think if I can do it on my own, I will learn the details and understand what it works. Moreover, I want to design the architecture like moving the time-consuming computation into c++, creating a user interface that users can easily use this program, and then merge it together.
+
+
+## Implement
+
+### 1. Upload several images with different exposures 
+=> implement user interface
+
+### 2. Align Images
+To prevent ghosting artifacts, we need to align these images.
+
+=> implement alignMTB function
+
+### 3. Estimated the Camera Response Function
+By using CRF(Camera Response Function) recovery algorithm, we can estimate the CRF from these images by removing the "bad" pixels (too dark or too bright ), and averaging the remaining "good" pixels base on the exposure time.
+
+=> implement calibrateDebevec(CRF in OpenCV) 
+
+### 4. Merge Images together
+Once the CRF has been estimated, we can merge the exposure images into one HDR image.
+
+=> implement MergeDebevec(merge in OpenCV) 
+
+### 5.Tone mapping
+Converting a High Dynamic Range (HDR) image to a 24-bit image while preserving as much detail as possible for display purposes. There are many methods that provide tone mapping, I decide to implement one or two method depend on the time.
+These methods also have some parameters that can adjust, so we need to find the best parameters combination.
+
+=> implement TonemapDrago
+
+=> implement TonemapReinhard
+
+=> find the best parameters combination
+
+
+## Unittest
+- test the function correctness
+- test the user interface 
```

Comment:
Please describe, probably in a previous section, what UI you are envisioning, and update the means of testing here accordingly.

Reply:
I think that maybe Unit Test on UI is not necessary, so I delete it.
Unit Test will focus on the main functions.